packages/react-menu/src/MenuGroupHeader/useMenuGroupHeaderStyles.ts

```diff
@@ -0,0 +1,21 @@
+import { makeStyles, ax } from '@fluentui/react-make-styles';
```

Comment:
q: what is `ax` ?

Reply:
@Hotell FYI -> [make-styles/src/ax.ts](https://github.com/microsoft/fluentui/blob/master/packages/make-styles/src/ax.ts)

Reply:
A function that performs a merge of classes, similar to `classnames` packages, but handles also atomic output from `makeStyles()`.

```tsx
classnames(makeStyles(), makeStyles()) // ❌ will simply concat classes, result can contain duplicated definitions
ax(makeStyles(), makeStyles()) // ✅ will properly merge outputs
```

Reply:
`a -> atomic` right ? 
**out of topic** TBH I'm not fan of "smart abbreviations" (consumer can just guess what it might be right?), I understand it would be a breaking change though. I'd suggest to update docs for it - to explicitly state what the abbreviation means.

Reply:
> `a -> atomic` right ?
> **out of topic** TBH I'm not fan of "smart abbreviations" (consumer can just guess what it might be right?)

Yep

The `cx` abbreviation comes from `classnames` package (6.5m downloads/week, https://github.com/JedWatson/classnames/issues/157), Emotion also uses it: https://emotion.sh/docs/@emotion/css#cx. `ax` was chosen to explicitly differentiate it from `cx`, naming was taken from [`compiled`](https://compiledcssinjs.com/docs/class-names).